test_unstructured_ingest/unit/test_enhanced_dataclass.py

```diff
@@ -0,0 +1,36 @@
+import json
+from dataclasses import Field, dataclass, fields
+
+from unstructured.ingest.enhanced_dataclass import EnhancedDataClassJsonMixin, enhanced_field
+from unstructured.ingest.enhanced_dataclass.dataclasses import EnhancedField
+
+
+@dataclass
+class AuthData(EnhancedDataClassJsonMixin):
+    username: str
+    password: str = enhanced_field(sensitive=True)
```

Comment:
Can you add number, list and dict as well, just for peace of mind?

Reply:
What test case are you trying to validate?